test/rpc/test_rpc_misc.py

```diff
@@ -200,6 +201,59 @@ def test_get_events_wrong_blockid_type():
     assert resp["error"]["code"] == PredefinedRpcErrorCode.INVALID_PARAMS.value
 
 
+@pytest.mark.usefixtures("run_devnet_in_background")
+@pytest.mark.parametrize(
+    "run_devnet_in_background",
+    [PREDEPLOY_ACCOUNT_CLI_ARGS],
+    indirect=True,
+)
+def test_get_events_continuation_token():
+    """
+    Test RPC get_events returning continuation token.
+    """
+    deploy_info = deploy(EVENTS_CONTRACT_PATH)
+    for i in range(3):
+        invoke(
+            calls=[(deploy_info["address"], "increase_balance", [i])],
+            account_address=PREDEPLOYED_ACCOUNT_ADDRESS,
+            private_key=PREDEPLOYED_ACCOUNT_PRIVATE_KEY,
+        )
+    resp = rpc_call(
+        "starknet_getEvents",
+        params=create_get_events_rpc(create_block_from_0(3))["params"],
+    )
+    assert len(resp["result"]["events"]) == 3
+    assert "continuation_token" not in resp["result"]
+
+    resp = rpc_call(
+        "starknet_getEvents",
+        params=create_get_events_rpc(create_block_from_0(1))["params"],
+    )
+    assert len(resp["result"]["events"]) == 1
+    assert resp["result"]["continuation_token"] == "1"
+
+    resp = rpc_call(
+        "starknet_getEvents",
+        params=create_get_events_rpc(create_block_from_0(1, "1"))["params"],
+    )
+    assert len(resp["result"]["events"]) == 1
+    assert resp["result"]["continuation_token"] == "2"
+
+    resp = rpc_call(
+        "starknet_getEvents",
+        params=create_get_events_rpc(create_block_from_0(1, "2"))["params"],
+    )
+    assert len(resp["result"]["events"]) == 1
+    assert "continuation_token" not in resp["result"]
```

Comment:
Wonder if using `@pytest.mark.parametrize` instead of multiple lines of basically duplicated code would make it more readable.

Reply:
Done like suggested below